tests/core_proxy/core_proxy.cpp

```diff
@@ -80,8 +80,7 @@ int main(int argc, char* argv[])
 
 
   po::options_description desc("Allowed options");
-  // tools::get_default_data_dir() can't be called during static initialization
```

Comment:
No obvious consequences after actual moving this call to the static initialization in option declaration, but I may be missing something.